### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by briancavanaugh**

### CODE REVIEW

File: CI/Pushpay/credit-card-people-export.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	JOIN [MinistryPlatform].[dbo].[Programs] p ON p.Program_ID = rg.Program_ID
+	JOIN [MinistryPlatform].[dbo].[Recurring_Gift_Frequencies] rgf ON rgf.Frequency_ID = rg.Frequency_ID
+	JOIN [MinistryPlatform].[dbo].[Donor_Accounts] da ON rg.Donor_Account_ID = da.Donor_Account_ID
+	LEFT OUTER JOIN [MinistryPlatform].[dbo].[Recurring_Gift_Days] rd ON rd.Day_Of_Week_ID = rg.Day_Of_Week_ID
+WHERE
+	rg.Start_Date <= GETDATE()
+	AND rg.End_Date IS NULL
+	-- AND da.Account_Type_ID in (1, 2) -- Only export bank accounts (Checking, Savings), since we are not importing Credit Card recurring gifts
+	AND da.Account_Type_ID = 3 -- Only export credit card
+	AND rg.Subscription_ID LIKE 'sub_%' -- only way I can think of to identify stripe gifts vs. pushpay gifts
```
Comment (jpc11623) [inline CI/Pushpay/credit-card-people-export.sql]: Do we need to filter for guest givers for CC recurring gifts?

Comment (briancavanaugh) [reply CI/Pushpay/credit-card-people-export.sql]: That would make sense.  How would we identify guests?  No email?

Comment (jpc11623) [reply CI/Pushpay/credit-card-people-export.sql]: guest giver nickname is "guest giver" - I have the filter in the query I sent you earlier.

Comment (briancavanaugh) [reply CI/Pushpay/credit-card-people-export.sql]: Got it..... I will update

**Pull request merged**